docs/wiki/ParameterFileTokens.md

```diff
@@ -71,79 +63,77 @@ The Key Vault here is enabled by adding a Secret to GitHub called `PLATFORM_KEYV
+3- The Platform Key Vault is then deployed using the Key Vault Module and the dependency [Platform Key Vault Parameter File](https://github.com/Azure/ResourceModules/blob/main/utilities/dependencies/Microsoft.RecoveryServices/vaults/parameters/platform.parameters.json).
 
-  > You need to have permissions to read and create secrets in the provisioned Key Vault so that you can upload custom tokens. The [Key Vault Secrets Officer](https://docs.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#key-vault-secrets-officer) role is an example that allows you to perform data plane operations on the Key Vault.
-
-4- Once the Key Vault is deployed, the 'Set Local Tokens in Key Vault' task optionally runs to sync Local Custom Parameter File Tokens stored in the [Settings.json](https://github.com/Azure/ResourceModules/blob/main/settings.json)
+> This Key Vault uses [Azure AD RBAC](https://docs.microsoft.com/en-us/azure/key-vault/general/rbac-guide?tabs=azure-cli) for the permissions instead of [access policies](https://docs.microsoft.com/en-us/azure/key-vault/general/assign-access-policy?tabs=azure-portal). It will automatically provide the Deployment Service Principal with the [Key Vault Administrator](https://docs.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#key-vault-administrator) Role so that it can read and set secrets. </br>
 
-  > You can disable the sync process from Local Tokens (in Source Control) to Remote Tokens (in Key Vault) by modifying the `keyVaultSecretNamePrefix` property in the [Settings.json](https://github.com/Azure/ResourceModules/blob/main/settings.json) file and setting it to `false`.</br>
-  > This step/task is optional and can be used to share tokens across different repositories.
+  > You need to have permissions to read and create secrets in the provisioned Key Vault so that you can upload custom tokens. The [Key Vault Secrets Officer](https://docs.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#key-vault-secrets-officer) role is an example that allows you to perform data plane operations on the Key Vault.
 
-### Get Parameter File Tokens
+### How Tokens Are Replaced in a Parameter File
 
  ---
 **Note**: This Step is always enabled even if you do not use Local/Remote Custom Parameter File Tokens. Default Tokens are always required and are Pester Tested if they do not exist in a Parameter file where they should.
```

Comment:
-> `Default Tokens are always required and are alerted upon by Pester if they do not exist in the parameter file they should`